pom.xml

```diff
@@ -1327,6 +1331,10 @@
           <exclusion>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-reload4j</artifactId>
+    </exclusion>
+    <exclusion>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
```

Comment:
I believe it was complaining to either remove it from `hadoop-hdfs` or `hadoop-mapreduce-client-core`. So, this exclusion may not be required.